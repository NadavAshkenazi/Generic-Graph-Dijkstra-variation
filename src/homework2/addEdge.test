CreateNode n1 2
CreateGraph X
CreateNode n2 3
AddNode X n1
AddNode X n2
AddEdge X n2 n1 
ListChildren X n2
