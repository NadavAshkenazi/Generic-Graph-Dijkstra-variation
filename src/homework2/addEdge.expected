created node n1 with cost 2
created graph X
created node n2 with cost 3
added node n1 to X
added node n2 to X
added edge from n2 to n1 in X